about.md

```diff
@@ -0,0 +1,59 @@
+
+### Software Development Tooling
+PELUX comes with tooling to support the development of services. The Software Development Kit (SDK) provides good practices, documentation and tooling on how to create Linux-based automotive systems. This enables any developer or designer to achieve their best in their own domain of competence. We also make sure knowledge is transparently shared with all teams.
+
+### Integrate Continuously
+PELUX is designed for a development process built on agile development methods and continuous integration and delivery strategies. Using technologies such as Yocto, Git and Jenkins lets you manage complex projects with a high degree of transparency, providing immediate feedback into your agile development process.
+
+### Vehicle Integration
+One of the key challenges of bringing consumer-grade technology to automotive applications is efficiently integrating it into electrical architectures. With expert knowledge from automotive-specific networking and diagnostics technologies, PELUX can be configured to support an automotive network such as Ethernet AVB and provide support for Adaptive AUTOSAR&reg;. Luxoft Automotive provides MLB connectivity and also launched the Open Source GENIVI&reg; EAVB Reference Implementation project.
+
+### Professional Services
+Whether it is helping you evaluate your architectural requirements or developing and deploying your product to the market, Luxoft Automotive provides a key mix of top industry experts and Open Source Linux engineers that will get the job done. Our competence ranges from pixel to silicon, enabling us to support your projects with a holistic view on how to design and deliver the end product. With many years of experience, Luxoft Automotive is your best partner for supporting both small- and large-scale projects.
+
+## Additional Features
+- GENIVI&reg;-compliant Linux-based platform derived from poky
+- Luxoft Software Factory including continuous integration and test frameworks based on Yocto, Git and Jenkins 
+- Reference integration with Qt Automotive Suite to demonstrate possible integration with other UI frameworks
+- Software Development Kit (SDK) for platform management, service development and component integration
+- Multiple SoC BSPs available
+- Publicly accessible CI
+- Optional support for EAVB networking and MLB connectivity
+
+## Benefits
+- Outstanding UX performance and design freedom with powerful integration of third-party functionality, supporting unified APIs and a bundled SDK
+- Supports agile development for both platform and applications through Luxoft Software Factory
+- Access to large ecosystems of content and developers through Qt, Populus, HTML and Linux
+
+## Platforms Using PELUX
```

Comment:
I'd say it's more "Platforms currently supported in PELUX" than that the platforms are using PELUX.

Reply:
This text comes from the PELUX factsheet which is under development, and I'm not keen to change this text right now. I will forward your comment to the "factsheet responsible" and we'll see if it will be changed or not.